packages/duckdb-server-rust/src/app.rs

```diff
@@ -42,7 +42,7 @@ async fn handle_post(
 
 pub fn app() -> Result<Router> {
     // Database and state setup
-    let db = ConnectionPool::new(":memory:", 16)?;
+    let db = ConnectionPool::new(":memory:", 1)?; // TODO: we can only use one connection since temp tables are scoped per connection
```

Comment:
Once #519 lands data cube indexes will no longer be temp tables. But other Mosaic exec calls may make temp tables, so I don't think we can expand the connection pool...

Reply:
Yes, will leave this at 1 for now. 

What are your thoughts on not using temp tables at all anymore? We could change all the examples to write tables into a separate namespace so they are still easy to clean up.